src/features/course-main/course-main.tsx

```diff
@@ -37,7 +37,7 @@ export const CourseMain = ({ courseName, type }: CourseMainProps) => {
         </div>
         <div className="info">
           <SectionLabel label={label} />
-          <Title text={altTitle || title} />
+          <Title text={`${altTitle || title} Course`} />
```

Comment:
I think it will be better to remove all 'hardcoding' from the components

Reply:
I think it would be a good idea to refactor these components. I can see what can be done when doing this task https://github.com/rolling-scopes/site/issues/143

Reply:
It's doesn't relevant #143  task. You can create an issue for refactoring, but it is not good practice - write `wrong` code, merge it and then refactor it for `good` code